aimmo-game/tests/test_simulation/test_pickups.py

```diff
@@ -44,18 +48,72 @@ def test_effect_added(self):
 class TestHealthPickup(_BaseCases.BasePickupTestCase):
     pickup_class = pickups.HealthPickup
 
+    def test_invulnerability_is_applied(self):
+        self.apply_pickup()
+        self.assertIsInstance(self.pickup, pickups.HealthPickup)
+
     def test_health_increases(self):
         self.apply_pickup()
         self.assertEqual(self.avatar.health, 8)
 
     def test_serialise(self):
         self.assertEqual(self.pickup.serialise(), {'type': 'health', 'health_restored': 3})
 
+    # Test that the health pickup with a custom health gain value is applied correctly. We do not check
+    # for restraints (ie. less than 0 or more than 100) in the following tests:
+    def test_health_increases_custom_positive_5(self):
+        self.avatar.health = 10
+        self.set_custom_pickup(5)
+        self.apply_pickup()
+        self.assertEqual(self.avatar.health, 15)
+
+    def test_health_increases_custom_positive_50(self):
+        self.avatar.health = 10
+        self.set_custom_pickup(50)
+        self.apply_pickup()
+        self.assertEqual(self.avatar.health, 60)
+
+    # Checking that the health gain argument raises exceptions in correct scenarios.
+    def test_health_argument_raises_exception_one(self):
```

Comment:
no descriptive enough - please replace one two and three with a short description of what the scenario is

Reply:
... or reduce to one test the for same responsibility.